src/state/markets/fifo.rs

```diff
@@ -822,7 +822,8 @@ impl<
 
         if order_packet.is_expired(current_slot, current_unix_timestamp) {
             phoenix_log!("Order parameters include a last_valid_slot or last_valid_unix_timestamp_in_seconds in the past, skipping matching and posting");
-            return None;
+            // Fail softly if the order is expired
```

Comment:
prefer "Don't fail loudly" or something like "Don't fail the entire tx, but this order doesn't make it to the book"